File: Facultate/Anul 2/Sem 2/MPP/Laburi/JavaTema4/JavaTema4/Server/src/main/java/ServerMain.java

import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;
import server.ObjectConcurrentServer;
import utils.Props;
import java.util.Properties;

public class ServerMain {
    private static final Logger logger = LogManager.getLogger(ServerMain.class);

    public static void main(String[] args) throws Exception {
        try {
            int port = 5555;
            Properties prop = Props.getProperties();
            System.out.println(prop);
            ObjectConcurrentServer server = new ObjectConcurrentServer(port);
            server.start();
        } catch (Exception e) {
            logger.error("Failed to start server", e);
            System.exit(1);
        }
    }
}

File: Facultate/Anul 2/Sem 2/MPP/Laburi/CSharpLab_3/Server/ServerMain.cs
using Server.server;

namespace Server;

public class Program
{
    public static void Main(string[] args)
    {
        try
        {
            int port = 5555;

            var server = new ObjectConcurrentServer(port);
            server.Start();
        }
        catch (Exception)
        {
            Environment.Exit(1);
        }
    }
}